src/stories/index.stories.js

```diff
@@ -342,6 +347,33 @@ storiesOf('Map', module)
       },
     }
   })
+
+  .add('add and remove layers', () => {
+    return {
+      template: dynamicLayersTemplate,
+      data: () => ({
+        style: 'mapbox://styles/global-data-viewer/cjtss3jfb05w71fmra13u4qqm',
```

Comment:
this is from a specific project? is it useful to relate that to this general package, even if only in  a story?